.github/workflows/main.yml

```diff
@@ -17,7 +17,11 @@ jobs:
       run: |
         python setup.py install
         echo SyferTest successfully installed.
+    - name: Install test dependencies
+      run: |
+        pip install ".[test]"
+        echo Test dependencies successfully installed.
```

Comment:
I would combine all of those lines into one section to avoid redundancy in SyferText install commands:

```
name: Install SyferText with test dependencies
      run: |
        pip install .[test]
        echo SyferText with test dependencies successfully installed.
```

Reply:
Done.